pkg/ovsdb/ovsdb.go

```diff
@@ -30,26 +32,85 @@ const (
 	ovsTable    = "Open_vSwitch"
 )
 
-// Bridge Model of the Bridge table
+type (
+	// BridgeFailMode defines failure mode
+	BridgeFailMode = string
+	// BridgeProtocols defines OpenFlow Protocol version
+	BridgeProtocols = string
+)
+
+const (
```

Comment:
Please keep only values that are used in the project

Reply:
Ok, fixed.